api_tests/requests/views/test_request_list_create.py

```diff
@@ -125,7 +126,10 @@ def test_filter_by_machine_state(self, app, project, noncontrib, url, admin, nod
 @pytest.mark.django_db
 class TestPreprintRequestListCreate(PreprintRequestTestMixin):
     def url(self, preprint):
-        return f'/{API_BASE}preprints/{preprint._id}/requests/'
+        preprint_id = preprint._id
+        if VersionedGuidMixin.GUID_VERSION_DELIMITER in preprint._id:
+            preprint_id = preprint._id.split(VersionedGuidMixin.GUID_VERSION_DELIMITER)[0]
+        return f'/{API_BASE}preprints/{preprint_id}/requests/'
```

Comment:
Instead of modifying the test to use base guid, we should update the api code to support versioned guid.